### DISCUSSION TITLE
Getting:  connection_shutdown while stress testing on single broker

### DISCUSSION THREAD

**Discussion by jayaraj** [emqx/emqx#11693] [General]
### What happened?

Hi,
    While stressing using mqtt-stresser on a single broker, I am getting  these errors

`2023-09-23T14:15:05.087789+00:00 [error] supervisor: 'esockd_connection_sup - <0.2819.0>', errorContext: connection_shutdown, reason: #{max => 1000,reason => message_queue_too_long,value => 1847}, offender: [{pid,<0.11024.8>},{name,connection},{mfargs,{emqx_connection,start_link,[#{enable_authn => true,limiter => undefined,listener => {tcp,default},zone => default}]}}]`

I am able to make 50000 connections too, but my client is getting disconnected because this message queue size is there a way to increase this queue size

### What did you expect to happen?

How to increase message queue size

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
# paste output here
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Log files

<details>

</details>

**Comment by qzhuyan** [comment]
look for `force_shutdown` in https://www.emqx.io/docs/en/v5.0/configuration/configuration-manual.html#performance-optimization.
